editor/src/components/text-editor/text-editor.spec.browser2.tsx

```diff
@@ -417,3 +496,13 @@ export var storyboard = (
   </Storyboard>
 )
 `)
+
+const emptyProject = formatTestProjectCode(`import * as React from 'react'
+import { Storyboard } from 'utopia-api'
+
+
+export var storyboard = (
+  <Storyboard data-uid='sb'>
```

Comment:
note: this is intentionally left "ugly" to make sure the editor actually does its thing and cleans up the tree